package.json

```diff
@@ -3,7 +3,8 @@
   "version": "8.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npx concurrently \"yarn next dev\" \"yarn theme:watch\"",
```

Comment:
This is very much a personal preference. concurrently is not added to the project as a dev dependency, but run remotely. The only requirement that would be prompted in the console is for the user to install it to their local machine. (This package is used in Chakra projects)

This command ensures that the dev environment and the Chakra CLI run in parallel without the user needing to remember an extra step.

Can certainly revert this if this is too much.

Reply:
Agreed, this is more of a personal preference. In that sense, I'd prefer to keep the `dev` script as clean as possible.

However, I do see the benefit of running the Chakra script in parallel. We could add a comment in the readme or docs to let devs know how this works.

cc @wackerow @nhsz @corwintines

Reply:
Agree that this should be opt-in with another script, not default `npm dev` behavior, which should follow the standard.

Reply:
I have reverted the `dev` script.

Would you want some comment added in this PR?